### ISSUE TITLE
Memory out of bounds w/ Jaro

### ISSUE THREAD

**Issue by RevCBH** [vporoshok/muzzy#1]
I haven't had a chance to dive any deeper yet, but found a minimal repro for a runtime panic case when using Jaro or JaroWinkler:

```go
package main

import "github.com/vporoshok/muzzy"

func main() {
  muzzy.Similarity("mousa mohammed abu marzook", "abu marzook mousa mohammed", muzzy.JaroWinkler, 0)
}
```

**Comment by vporoshok** [comment]
Hi! Thank you for this report. I'll try to fix it this weekend

**Comment by RevCBH** [comment]
Cool. If you don't have time, I'll take a crack at it. For my immediate needs, I was able to fall back to another algo w/ reasonable results. Thanks for the library!

**Comment by vporoshok** [comment]
fixed in v0.2.1
